src/app/layout.tsx

```diff
@@ -4,13 +4,16 @@ import { Roboto } from 'next/font/google'
 import Header from '../components/Header/Header'
 import { Footer } from '../components'
 
+//TODO for create fonts configuration variable.
 const roboto = Roboto({
+  weight: ['400', '500', '700', '900'],
+  style: ['normal', 'italic'],
   subsets: ['latin'],
-  weight: '400',
+  display: 'swap',
 })
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
+  title: 'iPlayed.games',
   description: 'Generated by create next app',
```

Comment:
can you copy description for  IPLAYED ? i guess there is the data goes for SEO.

or can you add a TODO: for adding description